### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by briferz**
bla bla bla

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+** Simple Fast-API Project **
```
Comment (LolaMembrila) [inline README.md:1]: Buena idea los asteriscos dobles :)

**Review by LolaMembrila — CHANGES_REQUESTED** [review]
Sería mejor cambiar la línea "This is a test project" por "This is the best test project"

File: README.md (hunk 1 above)
Comment (briferz) [reply README.md:1]: Ok

Comment (briferz) [reply README.md:1]: I think this is pretty safe to deploy because ...

**Review by LolaMembrila — APPROVED** [review]
LGTM

**Pull request merged**